.drone.yml

```diff
@@ -23,9 +23,10 @@ pipeline:
   # deployment to staging environment
 
   build_stg:
-    image: alpine:3.6
+    image: node:8-alpine
     environment:
       - BASE_URL="//blog-staging.srcd.run/"
+      - ENVIRONMENT="staging"
```

Comment:
If adding this, maybe BASE_URLs can be defined somewhere else and removed from here?